quickstarts/microsoft.maintenance/maintenance-configuration-assignments/azuredeploy.parameters.json

```
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "assignmentName": {
+            "value": "ABCD-maintenance-configassignment"
+        },
+        "vmName": {
+            "value": "/subscriptions/ba45ac97-05ea-49fa-87e2-554cc87532bf/resourcegroups/abcdefgh/providers/microsoft.compute/hostgroups/abcdef/hosts/host_name"
```

This should just be the vmName
```suggestion
            "value": "host_name"
```